### PR TITLE
GitHub action to get oidc token

### PR DESCRIPTION
**Pull request by Sawthis**
**Description**

Changes proposed in this pull request:

- Example of workflow for getting and debugging OIDC tokens

**Related issue(s)**

https://github.com/kyma-project/test-infra/issues/9976

### CODE REVIEW

File: .github/workflows/get-oidc-token.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      id-token: write # This is required for requesting the JWT token
+      contents: read  # This is required for actions/checkout
+    runs-on: ubuntu-latest
+    name: A job to get OIDC token and say hello
+    steps:
+      # This action is used to get the JWT token from the OIDC provider and set is as an output and an environment variable
+      - uses: Sawthis/expose-jwt-action@v1.0.4
+        name: Get JWT token
+        with:
+          aud: 'https://github.com/github'
```
Comment (akiioto) [inline .github/workflows/get-oidc-token.yml:17]: Adjust by choosing either 'aud' or 'audience' for consistency. (Line 28)

Comment (Sawthis) [reply .github/workflows/get-oidc-token.yml]: done

**Review by akiioto — APPROVED** [review]

File: .github/actions/expose-jwt-action/README.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# View GitHub JWT Token
+
+This action takes an optional input for the `audience` value and outputs a GitHub signed JWT. 
+
+## Inputs
```
Comment (IwonaLanger) [inline .github/actions/expose-jwt-action/README.md:5]: Do we need a plural form here?

File: .github/actions/expose-jwt-action/README.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+Default: ``
+
+## Outputs
+
+## `jwt`
+
+The JSON Web Token signed by GitHub Action.
+
+## Example usage
```
Comment (IwonaLanger) [inline .github/actions/expose-jwt-action/README.md:19]:
```suggestion
## Example Usage
```

File: .github/actions/expose-jwt-action/README.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+## Inputs
+
+## `audience`
+
+The audience field in the JWT.
+
+Default: ``
+
+## Outputs
```
[the remark this answers is not in the document]
Comment (Sawthis) [reply .github/actions/expose-jwt-action/README.md:13]: It's from example README: https://docs.github.com/en/actions/creating-actions/creating-a-javascript-action#creating-a-readme

File: .github/actions/expose-jwt-action/README.md (hunk 2 above)
Comment (Sawthis) [reply .github/actions/expose-jwt-action/README.md:5]: It's from example README: https://docs.github.com/en/actions/creating-actions/creating-a-javascript-action#creating-a-readme

**Review by akiioto — APPROVED** [review]

**Pull request merged**